.travis.yml

```diff
@@ -1,27 +0,0 @@
-language: python
```

Comment:
From what I'm seeing, we have not completed a Travis build in the last two years, so we should be able to get rid of this file and presumably the integration altogether.
<img width="292" alt="Screenshot 2023-06-26 at 11 34 39 AM" src="https://github.com/lucyparsons/OpenOversight/assets/5885605/a197a634-4d20-481b-9350-b0e5d1e3e1ae">